mitsurunner.yaml

```diff
@@ -11,17 +11,19 @@ esphome:
   on_boot: # Laitteen bootatessa varmista, että sulatus ei käynnisty päälle kytkemällä se pois päältä ja käynnistä ajastus pakkosulatukselle
   - switch.turn_on: gpio_rele
   - script.execute: ajasta_pakkosulatus
-  - script.execute: kaynnista_minimi_aika
+#  - script.execute: kaynnista_minimi_aika
+  includes:
+    - constants.h
 
 # Globaalit muuttujat, joilla estetään esim. alle minimiajan tapahtuvat sulatukset.
 globals:
   - id: G_minimiaika_kulunut
     type: bool
-    restore_value: false
-    initial_value: 'false'
+    restore_value: yes
+    initial_value: 'true'
```

Comment:
Päätellään käynnistyksen lämpötiloista, mitä laite alkaa tehdä

Reply:
Tämän olitkin korjannut jo myöhemmällä commitilla. Hyvä

Reply:
Juuri näin. Jätetään restoret pois. Se ei toimi, koska ei tiedetä kauanko ollaan oltu "taju kankaalla".